www/src/components/PageEditBtn/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Hyperlink } from '~paragon-react';
+
+function PageEditBtn({ githubEditPath, ...props }) {
+  const pageEditUrl = `https://github.com/openedx/paragon/edit/master/src${githubEditPath}`;
+
+  const handlePageEditBtnClick = () => {
+    global.analytics.track('openedx.paragon.docs.page_edit.clicked');
+  };
+
+  return (
+    <Button
+      size="sm"
+      as={Hyperlink}
+      destination={pageEditUrl}
+      variant="tertiary"
+      onClick={handlePageEditBtnClick}
+      target="_blank"
+      rel="noopener noreferrer"
```

Comment:
nit: Is the `rel` attribute needed if we're using `Hyperlink` with `target="_blank"`?

Reply:
Yes, sure
It does not make sense